test/cri-containerd/test-images/nanoserver-gracefultermination-repro/latest/delayed-shutdown.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+func main() {
+	fmt.Println("Waiting for OS signal...")
+
+	signalChannel := make(chan os.Signal)
+	wait := make(chan int, 1)
```

Comment:
If you are just using a channel to wait on once, you can do `chan struct{}`, and `close(wait)` instead of sending to the channel. If you want to send/receive multiple times, you can still do `chan struct{}` and use `wait <- struct{}{}` to send. This is probably a little more efficient than sending an `int`.

Reply:
Not a big deal to change it here if you don't want to, just letting you know.

Reply:
Sure will keep this in mind